test/mock_windows.uts

```diff
@@ -167,21 +167,24 @@ test_read_routes6_windows()
 from scapy.config import conf
 assert conf.prog.powershell != None
 
-= Test all VBS fallback
+= Store powershell results
 import mock
 from scapy.config import conf
 
 ps_ip = get_ip_from_name(conf.iface.name)
+ps_ip
```

Comment:
What are you testing here?

Reply:
This is very useful when those tests fail. It does not really test anything, but is useful to quickly check what’s failing (when it fails). I’ve splitted the tests into several parts so that it’s easier to debug.

Those tests are very depending on the windows version, as VBS is not commonly used. It’s useful to compare the expected values to the one received

Reply:
Got it.